### DISCUSSION TITLE
Apple M1 mac mini working!

### DISCUSSION THREAD

**Discussion by masinter** [Interlisp/medley#105] [Show and tell]
I had to use MacPorts after first manually removing old x86 libraries and files left by XQuartz install.
Because it was handy I timed the running of the opcode tests. Looks like almost twice as fast as i7700 3.6ghz windows 

(the mac mini is on the left)

[Screenshot 2020-12-22 225136](https://user-images.githubusercontent.com/1116587/102968615-0b9e0800-44a9-11eb-8ca2-3af7b1f99a43.png)

**Comment by waywardmonkeys** [comment]
How do you get that trace output?

**Reply by masinter** [reply]
(PROGN (IL:BKSYSBUF " ") (IL:TIME (LOAD "{CORE}<TESTS>OPCODES.DFASL")))
REDO

**Comment by waywardmonkeys** [comment]
Small correction after trying that ... `{CORE}<TESTS>OPCODES.DFASL` is the right path (at least for the `test.vm`).
